src/traffic_server/InkAPI.cc

```diff
@@ -8274,6 +8300,30 @@ TSHttpTxnConfigIntGet(TSHttpTxn txnp, TSOverridableConfigKey conf, TSMgmtInt *va
 
   *value = conv->get_int(src);
 
+  /* Set the value based on key for these, we either want sec or msec */
```

Comment:
I think putting this special code in two places is not the best plan. I think it should be handled via the converters. I will try to provide some example code.